pkg/specs/replicatedapp/resolver.go

```diff
@@ -165,7 +167,24 @@ func (r *resolver) resolveCloudRelease(selector *Selector) (*ShipRelease, error)
 	debug.Log("phase", "load-specs", "from", "gql", "addr", client.GQLServer.String())
 	release, err := client.GetRelease(selector)
 	if err != nil {
-		return nil, err
+		if selector.InstallationID == "" {
+			debug.Log("event", "spec-resolve", "from", selector, "error", err)
+			fmt.Printf("Please enter your license to continue: ")
```

Comment:
we use the mitchellh/cli components for prompting elsewhere in this project. 

They are a little more succint, and make this code easily testable

Reply:
Cool, I'll find that to copy it!